### DISCUSSION TITLE
Customize primary ip logic

### DISCUSSION THREAD

**Discussion by ShmoeTheHo** [bb-Ricardo/netbox-sync#153] [Ideas]
Can we get the ability to define what names are looked for on virtual port groups to define the primary ip connectivity? Right now its "management" or "mgmt" (May have the abbreviation characters wrong). Or maybe add the word "Utility" or "Utilities" to the logic.

Would it be possible to get a regex ability here? I used this setting and am now being asked to name the port group something else :( 

Anything would be much appreciated!

**Comment by bb-Ricardo** [comment]
Hi, good suggestion. I thought about this as well but haven't found any time to implement it.

Can you please open an issue and I will add it. With another config option 😂😭.
